.github/actions/publish/action.yml

```diff
@@ -48,9 +49,10 @@ runs:
             ghcr.io/goreleaser/goreleaser-cross:latest \
               -f /dev/null
         )"
-        docker exec --tty "$CONTAINER_ID" dpkg --add-architecture i386
-        docker exec --tty "$CONTAINER_ID" apt-get update
-        docker exec --tty "$CONTAINER_ID" apt-get install --no-install-recommends -y -q crossbuild-essential-i386
+        docker exec --tty "$CONTAINER_ID" mkdir -p /musl
+        docker exec --tty "$CONTAINER_ID" bash -c 'curl -fsSL https://musl.cc/aarch64-linux-musl-cross.tgz -o- | tar -zxvf - -C /musl --strip-components=1'
```

Comment:
This could be flaky and cause the release to fail - I'd add some `--retry` arguments to it to lower the chance.

Reply:
Added, we should probably create a custom base image for this build.